new-feature.txt

```diff
@@ -0,0 +1 @@
+Pierwszy plik w nowej galezi feature1
```

Comment:
hehehe 
Jacek Gmoch.

Reply:
w krajach zachodnich lampka wina to nic dziwnego do biadu.
Zdzisiu Krecina